db/schema.rb

```diff
@@ -21,6 +21,7 @@
     t.datetime "created_at", precision: nil, null: false
     t.datetime "updated_at", precision: nil, null: false
     t.index ["account_id"], name: "index_account_aliases_on_account_id"
+    t.index ["uri", "account_id"], name: "index_account_aliases_on_uri_and_account_id", unique: true
```

Comment:
What about having the index on `['account_id', 'uri']` so we can later drop `index_account_aliases_on_account_id`?